src/LogNormal/LogNormal.sol

```diff
@@ -82,7 +82,7 @@ contract LogNormal is PairStrategy {
         invariant =
             tradingFunction(reserves, totalLiquidity, getPoolParams(poolId));
         // todo: should the be EXACTLY 0? just positive? within an epsilon?
```

Comment:
Remove todo
```suggestion
```

Reply:
Dont commit suggestion actually, since we should run the formatter after deleting code comments